### PR TITLE
Getting Started Page

### PR DESCRIPTION
**Pull request by lmilechin**
Initial Getting Started page based on the SuperCloud one: https://supercloud.mit.edu/getting-started.

### CODE REVIEW

**Comment by lmilechin** [comment]
New Page: https://mit-orcd.github.io/orcd-docs-previews/PR/PR25/getting-started/

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+started with your ORCD system account. We provide this page as a
+convenient reference to get started. Each system has its own in-depth documentation which can be found on the [ORCD Systems](orcd-systems.md) page.
+
+Getting an Account
+------------------
+
+If you don't already have an account, follow the instructions for the system you are interested in:
+
+- Engaging and Satori: Login into the respective OnDemand Portal (below) using your MIT kerberos credentials. The system will then be prompted to create your account automatically. Wait couple minutes for the system to create all the pieces for your account before submitting your first job.
+    - Engaging: [https://engaging-ood.mit.edu]
```
Comment (lmilechin) [inline docs/getting-started.md:20]: Fix links

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+-->
+
+At this point you may want to do a test-run of your code. You always
+want to start small in your test runs, so you should choose a small
+example that tests the functionality of what you would ultimately like
+to run on the system. If your test code is serial and runs okay on a
+moderate personal laptop or desktop you can request an interactive
+session to run your code in by executing the command:
+
+`LLsub -i`
```
Comment (lmilechin) [inline docs/getting-started.md:263]: Update with codeblocks/tabs for different system

**Comment by shaohao99** [comment]
The link for [Logging into OpenMind].
https://github.mit.edu/MGHPCC/OpenMind/wiki/How-to-log-in-Openmind%3F

**Comment by shaohao99** [comment]
It would be good to make a pull-down section or a link to a new page for the Linux Command Line section.

File: docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+- [Satori](https://mit-satori.github.io/satori-getting-started.html#transferring-files)
+- [SuperCloud](https://supercloud.mit.edu/accessing-and-transferring-data-and-files)
+- [OpenMind](https://github.mit.edu/MGHPCC/OpenMind/wiki/How-to-transfer-files%3F)
+
+<!--
+TODO: Add engaging link
+-->
+
+
+Testing your Code
```
Comment (lmilechin) [inline docs/getting-started.md:249]: Change to "Running your First Job"

**Comment by lmilechin** [comment]
I added tabs for the sections where there are different information or links for the different systems. The tabs are synchronized so once you select one they will all be set to the same tab. Pretty cool!

Also updated the OpenMind Logging In link and put the Linux Commands in a collapsible box.

**Comment by shaohao99** [comment]
It looks very nice. I like the selectable and expendable tabs. I think it is good to publish.

**Comment by lmilechin** [comment]
Acceptable Use and Code of Conduct page also added: https://mit-orcd.github.io/orcd-docs-previews/PR/PR25/code-of-conduct/

**Comment by shaohao99** [comment]
Would it be good to make "Data Security and Privacy" and "Acceptable Use and Code of Conduct" sub-pages under a bigger title "ORCD policy" (or another similar name)? This will make the contents on the left column shorter, which is good for future topics. 

Other than that, I think it is good to publish.

**Comment by lmilechin** [comment]
Added acknowledgements page: https://mit-orcd.github.io/orcd-docs-previews/PR/PR25/acknowledgements/

**Review by christophernhill — APPROVED** [review]
Looks ready to go

**Comment by shaohao99** [comment]
Looks all good to me.

**Pull request merged**